src/value/owned.rs

```diff
@@ -801,22 +799,11 @@ mod value_tests {
         assert_eq!(e1.into(), e2.into());
     }
 
-    #[rstest(
-        container, length,
-        case::struct_(
-            ion_struct!{"greetings": "hello", "name": "Ion"},
-            2
-        ),
-        case::list(
-            ion_list!["greetings", 5, true],
-            3
-        ),
-        case::sexp(
-            ion_sexp!(5 true),
-            2
-        ),
-    )]
-    fn owned_container_len_test<I: Into<Element>>(container: I, length: usize) {
+    #[rstest]
+    #[case::struct_(ion_struct!{"greetings": "hello", "name": "Ion"}, 2)]
+    #[case::list(ion_list!["greetings", 5, true], 3)]
+    #[case::sexp(ion_sexp!(5 true), 2)]
+    fn owned_container_len_test<I: Into<Element>>(#[case] container: I, #[case] length: usize) {
```

Comment:
Compare to before the builder macros:
https://github.com/amazon-ion/ion-rust/blob/9019a2eaf972ad6e24c647fd54af00a1b022682b/src/value/owned.rs#L739-L764

The builder macros make this so nice!